images/cr_references.svg

```diff
@@ -0,0 +1,3 @@
text x="228" y="224" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">bootstrap</text></switch></g><rect x="318.24" y="390" width="136" height="60" rx="9" ry="9" fill="#000000" stroke="none" transform="translate(2,3)" opacity="0.25"/><rect x="318.24" y="390" width="136" height="60" rx="9" ry="9" fill="#d5e8d4" stroke="none" pointer-events="all"/><rect x="324.24" y="405" width="126.86" height="30" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 412px; margin-left: 388px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; "><font size="1" color="#4d4d4d"><span style="font-size: 14px">BareMetalHost</span></font></div></div></div></foreignObject><text x="388" y="424" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">BareMetalHost</text></switch></g><path d="M 454.24 420 L 683.96 278.34" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 688.43 275.59 L 684.31 282.24 L 683.96 278.34 L 680.63 276.28 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><rect x="470" y="330" width="171" height="30" fill="none" stroke="none" transform="rotate(-30,555.5,345)" pointer-events="all"/><g transform="translate(-0.5 -0.5)rotate(-30 551.5 338.07179676972453)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 338px; margin-left: 552px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; "><font style="font-size: 12px" color="#006666">consumerRef</font></div></div></div></foreignObject><text x="552" y="350" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">consumerRef</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
The infraref for machine/BMMachine is the wrong way around. And there is also an ownerref from BMMachine to Machine